example/cpp/unix/simple_tts.cpp

```diff
@@ -1,3 +1,5 @@
+// TODO: 書き方をsong.cppに寄せる
```

Comment:
これユーザーに間違えて伝わるかもしれないですねぇ

うーーーーん　一番最後に移すとか･･･？
いやまあこのままでも良いかも。

か、せめて更に前になにか書くとか･･･いや変わらないか。
まあでもこれくらいは書いても良いかも。
あと「書き方」だと何の書き方かわからないので明確にしてみました。

```suggestion
/**
 * テキスト音声合成を行う。
 */

// TODO: 実装スタイルをunix/song.cppに揃える
```